api/app/graphql/types/polymorphic_question_type.rb

```diff
@@ -10,6 +10,9 @@ def self.resolve_type(object, _context)
       when Question
         Types::QuestionType
       end
+      when TrueOrFalseQuestion
+        Types::TrueOrFalseQuestionType
+      end
```

Comment:
you need to add TrueOrFalseQuestionType to "possible_types". Take a look at line 4 on this file.

Reply:
Done